lib/msal-custom-auth/src/sign_up/auth_flow/error_type/SignUpError.ts

```diff
@@ -58,7 +58,7 @@ export class SignUpError extends AuthFlowErrorBase {
      * Checks if the challenge type is redirect (authentication method is not supported by by Microsoft Entra)
```

Comment:
nit: all over the comments in this class the word "by" is duplicated.
```suggestion
     * Checks if the challenge type is redirect (authentication method is not supported by Microsoft Entra)
```

Reply:
will remove this line.